tests/functional/spec/multiple-choice-other.spec.js

```diff
@@ -1,17 +1,98 @@
 import chai from 'chai'
 import {startQuestionnaire} from '../helpers'
+import OtherOptionsPage from '../pages/surveys/other_option/other-options.page'
+import OtherOptionsSummary from '../pages/surveys/other_option/otheroption-summary.page'
+import SummaryPage from '../pages/surveys/question.page'
 
 const expect = chai.expect
 
-describe('Multiple choice "other" option', function() {
+describe('Multiple choice Checkbox/Radio "other" option', function() {
 
-  before(function() {
+  it('Given an "other" option is available, when the user clicks the "other" option then other input should be visible', function() {
+
+    //Given
+    startQuestionnaire('0_checkbox_other.json')
+    //When
+    OtherOptionsPage.clickOther()
+
+    // Then
+    expect(OtherOptionsPage.otherTextFieldExits()).to.be.true
+  })
+
+
+  it('Given I have chosen other Option and not entered any text in the other field When I submit page then an error must be displayed  ', function() {
+
+    //Given
     startQuestionnaire('0_checkbox_other.json')
+    OtherOptionsPage.clickOther()
+
+    //When
+    OtherOptionsPage.submit()
+
+    //Then
+    expect(OtherOptionsPage.errorExists()).to.be.true
   })
 
-  it('Given an "other" option is available, when the user clicks the "other" option the other input should be visible', function() {
-    browser.click('[data-qa="has-other-option"]')
-    expect(browser.isVisible('[data-qa="other-option"]')).to.be.true
+  it('Given a error on the page for other field when I enter valid value and submit page Then error is cleared and move to summary page  ', function() {
+
+    //Given
+    startQuestionnaire('0_checkbox_other.json')
+    OtherOptionsPage.clickOther()
+    OtherOptionsPage.submit()
+    expect(OtherOptionsPage.errorExists()).to.be.true
+
+    //When
+    OtherOptionsPage.setTextInOtherField('Other Text')
+    .submit()
+
+    //Then
+    expect(OtherOptionsPage.errorExists()).to.be.false
+
   })
 
+  it('Given I have chosen other radio Option and entered hello in the other field When I submit page then on the summary screen I can see the text I entered displayed.  ', function() {
+
+    //Given
+    startQuestionnaire('0_radio_other.json')
+    OtherOptionsPage.clickOther()
+    expect(OtherOptionsPage.otherTextFieldExits()).to.be.true
```

Comment:
I don't think we need this check here as it will do the check implicitly when you set the text in setTextInOtherField